packages/plugin-core/src/components/lookup/NotePickerUtils.ts

```diff
@@ -81,6 +93,44 @@ export class NotePickerUtils {
     };
   }
 
+  static createNewWithTemplateItem({
+    fname,
+  }: {
+    fname: string;
+  }): DNodePropsQuickInputV2 {
+    const props = DNodeUtils.create({
+      id: CREATE_NEW_WITH_TEMPLATE_LABEL,
+      fname,
+      type: "note",
+      // @ts-ignore
+      vault: {},
+    });
+    const ABUserGroup = _2022_09_QUICKPICK_LABEL_HIGHLIGHT_TEST.getUserGroup(
+      SegmentClient.instance().anonymousId
+    );
+
+    let label: string;
+    if (ABUserGroup === QuickPickLabelHighlightTestGroups.label) {
+      label = LabelUtils.createLabelWithHighlight({
+        value: CREATE_NEW_WITH_TEMPLATE_LABEL,
+        highlight: {
+          value: "$(smiley) [New!] ",
+          location: "prefix",
+          expirationTime: DateTime.fromISO("2022-10-01"),
```

Comment:
At first I actually thought about tying this to the metadata so the expiration depends on the user actually seeing it, but I don't think that's a good way to approach feature highlighting here.

A feature is _new_ for a certain amount of period after the release. After that, it is just a regular feature that has been out for a while.

The problem we are tackling with this is "users are not aware of new features right away when they release", and after that, the problem statement changes to "users are not discovering our features". I would say those are two different problems that should be dealt in different ways.

I do understand where you're coming from though. But even when we change this so that it expires depending on the user seeing it or not, we still want the hard coded expiration date.
By doing that we are containing this to a specific time frame (our opinion on how long a feature belongs in _new_), eliminating the need to come back and clean it up (otherwise all new and returning users will be bombarded with features highlighted as _new_), and still basing the highlight on the fact that it was viewed by the user.

> i would increase the expiration time to a month. 
Got it. I think a month is a good time frame for us to use as default if we decide to use this highlighting feature more in the future as well.